lessons/_index.md

```diff
@@ -14,8 +14,8 @@ We currently have these lessons planned and we'll be adding more content over ti
 
 |Nr|Description|Language specific lesson|Stream/Recording|Contributions by
 |-|-|-|-|-
-|0|Prerequisites|[.NET Core](prerequisites-dotnet.md) , [TypeScript](prerequisites-ts.md)|[Oct 28th](https://youtu.be/5k35dlBAXxA)|Marc & Gwyneth (.NET Core) , Christian & Marc (TypeScript)
-|1|HTTP Trigger|[.NET Core](dotnet/http/http-lesson-dotnet.md) , [TypeScript](http-ts.md)|[Oct 28th](https://youtu.be/5k35dlBAXxA)|Marc & Gwyneth (.NET Core) , Christian & Marc (TypeScript)
+|0|Prerequisites|[.NET Core](prerequisites-dotnet.md) , [PowerShell](prerequisites-powershell.md), [TypeScript](prerequisites-ts.md)|[Oct 28th](https://youtu.be/5k35dlBAXxA)|Marc & Gwyneth (.NET Core) , Barbara (PowerShell), Christian & Marc (TypeScript)
```

Comment:
I've simplified this table and dropped the column with the direct links to the YouTube videos (so there's a merge conflict now 😬). I recommend you pull the latest changes and re-apply yours.

Reply:
well that was a fun merge 😀.
Added the links to the new table